### DISCUSSION TITLE
Q: 请问怎么禁用匿名评论？ || Q: How to disable anonymous comments?

### DISCUSSION THREAD

**Discussion by yenharvey** [walinejs/waline#2054] [Q&A] [answered]
### Describe the bug | 问题描述

请问能否在保留登录可选项的情况下禁用匿名登陆呢？ 设置requireMeta的话，右上角会出现一个匿名评论选项，用户点击就会随机生成用户昵称。[看了这个](https://github.com/orgs/walinejs/discussions/179)，但是这个答案里面的链接都404了

### Website URL | 问题网站

https://www.cutestgirl.cc/comments/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Can I disable anonymous login while keeping the login option? If requireMeta is set, an anonymous comment option will appear in the upper right corner, and the user's nickname will be randomly generated when the user clicks it. [Read this](https://github.com/orgs/walinejs/discussions/179), but the links in this answer are all 404

### Website URL | Problem Website

https://www.cutestgirl.cc/comments/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/reference/client/props.html#requiredmeta
